test/integration/narrow/error-pages.spec.js

```diff
@@ -1,115 +1,337 @@
+    });
+
+    describe('FORBIDDEN', () => {
+      let $;
+      const supportId = '12321432432';
+      beforeEach(async () => {
+        server.route({
+          method: 'GET',
+          path: '/forbidden',
+          handler: () => {
+            const error = new Error('Forbidden');
+            error.output = {
+              statusCode: statusCodes(FORBIDDEN),
+              payload: { message: supportId }
+            };
+            error.isBoom = true;
+            throw error;
+          }
+        });
+
+        const response = await server.inject({
+          method: 'GET',
+          url: '/forbidden'
+        });
+
+        $ = cheerio.load(response.payload);
+      });
+
+      it('should show correct message in title', () => {
+        expect($('title').text().trim()).toContain('The requested URL was rejected');
+      });
+
+      it('should show correct message in header', () => {
+        expect($('h1').text().trim()).toContain('Sorry, the requested URL was rejected');
+      });
+
+      it('should show correct message in paragraph', () => {
+        expect($('p.govuk-body').eq(0).text().trim()).toContain('Please try again later.');
+        expect($('p.govuk-body').eq(1).text().trim()).toContain(
+          'We have not saved your answers. When the service is available, you will have to start again.'
+        );
+        expect($('p.govuk-body').eq(2).text().trim()).toContain(`Your support ID is: ${supportId}`);
+        expect($('p.govuk-body').eq(3).text().trim()).toContain(
+          'Contact the RPA Helpline and follow the options for the Farming Investment Fund scheme if you have any questions.'
+        );
+        expect($('p.govuk-body').eq(4).text().trim()).toContain('Telephone: 0300 0200 301');
+      });
     });
 
-    await request(server.listener).get('/forbidden');
+    describe('INTERNAL_SERVER_ERROR', () => {
+      let $;
+      beforeEach(async () => {
+        server.route({
+          method: 'GET',
+          path: '/internal-error',
+          handler: () => {
+            const error = new Error('Internal Server Error');
+            error.output = { statusCode: statusCodes(INTERNAL_SERVER_ERROR) };
+            error.isBoom = true;
+            throw error;
+          }
+        });
 
-    expect(mockView).toHaveBeenCalledWith(statusCodes(FORBIDDEN).toString(), expect.anything());
-    expect(mockCode).toHaveBeenCalledWith(statusCodes(FORBIDDEN));
+        const response = await server.inject({
+          method: 'GET',
+          url: '/internal-error'
+        });
+
+        $ = cheerio.load(response.payload);
+      });
+
+      it('should show correct message in title', () => {
+        expect($('title').text().trim()).toContain('Problem with service');
+      });
+
+      it('should show correct message in header', () => {
+        expect($('h1').text().trim()).toContain('Sorry, there is a problem with the service');
+      });
+
+      it('should show correct message in paragraph', () => {
```

Comment:
Just a note to myself with snapshot tests we probably do not need these any more?

Reply:
To summarise - my idea is to keep these view tests more page behaviour and just the critical content orientated with the snapshot tests taking care of detailed content changes.

Reply:
Makes sense to me